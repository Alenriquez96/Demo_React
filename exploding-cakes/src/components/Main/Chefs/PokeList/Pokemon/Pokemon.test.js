import { render } from "@testing-library/react";
import React from "react";
import Pokemon from "./Pokemon";

describe("Pokemon", () => {
  test("matches snapshot", () => {
    render(<Pokemon />);
    expect(screen).toMatchSnapshot();
  });
});
