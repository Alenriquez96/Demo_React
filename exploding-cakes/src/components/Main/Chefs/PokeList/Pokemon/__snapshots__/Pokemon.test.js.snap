// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon matches snapshot 1`] = `Screen {}`;
